### DISCUSSION TITLE
Swapping runtime during compilation

### DISCUSSION THREAD

**Discussion by WhoAteDaCake** [mint-lang/mint#473] [Q&A] [answered]
Hello, is there any way to swap runtime, to a custom-defined one when compiling the program?

Something a long the lines of `mint build --runtime "./assets/my-runtime.js"` or mint start --runtime "./assets/my-runtime.js"

**Comment by gdotdesign** [comment, marked as answer]
There is no flag like that yet, but it's a good idea and would be easy to implement.

**Comment by gdotdesign** [comment]
It was implemented a while back.
